homeassistant/components/duotecno/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for duotecno integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from duotecno.controller import PyDuotecno
+from duotecno.exceptions import InvallidPassword
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_PORT): int,
```

Comment:
Does it have a default port we could set?

Reply:
No there is no hard default, most people use 5001 but this is not a standard, so i think its dificult to use this as a default.